.github/workflows/test.yml

```diff
@@ -9,14 +9,14 @@ jobs:
   linux-tests:
     strategy:
       matrix:
-        node: [10, 12, 14, 16, 18, 20]
+        node: [12, 14, 16, 18]
```

Comment:
Removed Node 20 for now because it was never working (i.e. you can't build/develop it successfully, but developers should still be able to use it in their project). 

CI did not catch this error because of incorrect setup. See below.

Reply:
I think we should create a ticket/github issue to track the work to eventually add this back in

Reply:
👍 ok I'll create the ticket.